Monika After Story/game/script-topics.rpy

```diff
@@ -4727,6 +4727,193 @@ label monika_write:
     m "Thanks for listening~"
     return
     
+init 5 python:
+    for key in ['feature','boobs','sight','personality']:
+        monika_topics[key].append('monika_standsout')
+    monika_random_topics.append('monika_standsout')
+
+label monika_standsout:
+    m 2a "Hey, [player]? What about me stands out the most to you?"
+    m "I know it's a random question. Maybe even a bit silly."
+    m 5a "But I'm dying to know."
+    menu:
+        m "So...?"
+        "Your beautiful, green eyes of course.":
+            m 4k "Aww, really?"
+            m 4e "Thank you, sweetheart."
+            m "I wish I could see your eyes."
+            m 1i "Actually, what color are your eyes?"
+            menu:
+                m 5a "I need to know now."
+                "Brown.":
+                    m 3b "Brown, huh?"
+                    m "I wish I could see them."
+                    m 2j "The stunning shade of rich mahogany. Or welcoming like that of a soft teddy bear."
+                    m 2a "I bet they're beautiful, my love."
+                    m 3j "Perhaps someday I'll have a chance to gaze deeply into them."
+              
+                 "Green.":
```

Comment:
Your menu choices for the colors here are not on the same indentation level. (Green, Blue,... )are all an extra space to the right.

Reply:
Fffffffffffffffffff
Move them back one tab?